main.py

```diff
@@ -2650,5 +2651,32 @@ async def recieve_vote(request):
     await claim_reward(user, channeley, type)
     return web.Response(text="ok", status=200)
 
+@bot.tree.command(name="fact", description="a fact")
+async def fact(message: discord.Interaction):
+    url = "https://catfact.ninja/fact"
+    facts = [
```

Comment:
lmao everything is capitalized god please fix this

Reply:
oh yea sorry 🙏